var/spack/repos/builtin/packages/debugedit/package.py

```diff
@@ -8,17 +8,18 @@
 
 class Debugedit(AutotoolsPackage):
     """
-    Debugedit was originally part of the rpm project, and is currently
-    being refactored to be provided outside of it. This source will
-    eventually be moved to sourceware or similar, as it is being maintained
-    by RedHat.
+    Debugedit was originally part of the rpm project, and now exists
+    as its own separate project and is maintained by RedHat.
```

Comment:
Can you make this also say something about what debug edit does?

Reply:
Yep.

Reply:
All set.